### PR TITLE
Add load balancing guidance for workloads in "Production environment" page

### PR DESCRIPTION
**Pull request by MissTipo**
## What?
I've ammended  production-environment to mention load balancing for workloads. It includes suggestion that the best solutions are external projects and the links to the pages with suggestions. See ticket #28060.

## Why?
Readers might want to know how to set up a cluster with load balancing implemented in a resilient and scalable way. See #28060 for more information.

## How?
This includes mentioning the need for load balancing for workloads, suggesting that the best solutions are external projects and linking a setting up external load balancer  modified to mention op-premises options and include a warning shortcode.

## Screenshots (optional)
0

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
/retitle Add load balancing guidance for workloads in "Production environment" page

File: content/en/docs/setup/production-environment/_index.md [hunk 1]
```diff
@@ -27,21 +28,28 @@ As you decide where you want your production Kubernetes environment to live
 on or hand to others, consider how your requirements for a Kubernetes cluster
 are influenced by the following issues:
 
-- *Availability*: A single-machine Kubernetes [learning environment](/docs/setup/#learning-environment)
+- _Availability_: A single-machine Kubernetes [learning environment](/docs/setup/#learning-environment)
```
Comment (windsonsea) [inline content/en/docs/setup/production-environment/_index.md:31]: `*` has the same effect as `_`. Is any special to perform these 50+ changes?

**Review by windsonsea — COMMENTED** [review]
It would be better to focus the issue fixes.

**Review by satyampsoni — COMMENTED** [review]
what is the value this PR is adding?

**Comment by kbhawkey** [comment]
Hello @MissTipo .
I don't think these changes are helpful. The existing formatting on the page is okay.
/close

**Pull request closed without merging**